go/vt/worker/instance.go

```diff
@@ -129,6 +130,9 @@ func (wi *Instance) setAndStartWorker(ctx context.Context, wrk Worker, wr *wrang
 	// one go function runs the worker, changes state when done
 	go func() {
 		log.Infof("Starting worker...")
+		span, ctx := trace.NewSpan(wi.currentContext, "work")
+		span.Annotate("extra", wrk.StatusAsText())
```

Comment:
vtworker status can be quite big. Might be excessively large to be an annotation, and its value is also somewhat opaque and non-specific which doesn't feel very annotation-like.